### PR TITLE
Added support to variable length string arguments longer than 128 characters.

### PR DESCRIPTION
**Pull request by MiranDMC**

### CODE REVIEW

File: source/CCustomOpcodeSystem.cpp [hunk 1]
```diff
@@ -400,40 +400,58 @@ namespace CLEO {
 	// read string parameter according to convention on strings
 	char *readString(CRunningScript *thread, char* buf = nullptr, BYTE size = 0)
 	{
+		if (size == 0) size = 0xFF; // max
```
Comment (x87) [inline source/CCustomOpcodeSystem.cpp:403]: 0xFF probably should be defined as a const MAX_STR_LEN

Comment (MiranDMC) [reply source/CCustomOpcodeSystem.cpp]: Right! Updated.

File: source/CCustomOpcodeSystem.cpp [hunk 2]
```diff
@@ -2285,12 +2302,12 @@ namespace CLEO {
 	//0ADF=2,add_dynamic_GXT_entry %1d% text %2d%
 	OpcodeResult __stdcall opcode_0ADF(CRunningScript *thread)
 	{
-		char *entryName;
-		char text[128];
-		char *str;
-		entryName = readString(thread);
-		str = readString(thread, text, sizeof(text));
-		GetInstance().TextManager.AddFxt(entryName, str);
+		char gxtLabel[8 + 1]; // and terminator character
```
Comment (x87) [inline source/CCustomOpcodeSystem.cpp:2305]: isn't the max length of a GXT key 8 (7 characters + null terminator)?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+## 4.4.5
+
+- added support of variable length arguments longer than 128 characters
+
```
Comment (x87) [inline CHANGELOG.md:4]: can you add this change under 4.4.4, we didn't release that one yet

Comment (MiranDMC) [reply CHANGELOG.md:4]: ok

File: source/CCustomOpcodeSystem.cpp (hunk 2 above)
Comment (MiranDMC) [reply source/CCustomOpcodeSystem.cpp]: Right. I thought opcodes know it is up to 8 without need for terminator. Fixed.

**Review by x87 — APPROVED** [review]

**Pull request merged**